tests/unit/test_guess_packager.py

```diff
@@ -0,0 +1,124 @@
+# Copyright Contributors to the Packit project.
+# SPDX-License-Identifier: MIT
+
+import subprocess
+from collections.abc import Iterable
+from pathlib import Path
+from unittest.mock import patch
+
+import pytest
+import rpm
+
+from specfile.changelog import guess_packager
+
+
+@pytest.fixture
+def clean_guess_packager(
+    tmp_path: Path, monkeypatch: pytest.MonkeyPatch
+) -> Iterable[None]:
+    """
+    Ensure a clean environment
+    """
+    # For $RPM_PACKAGER
+    monkeypatch.delenv("RPM_PACKAGER", False)
+    # Make sure git doesn't read existing config
+    monkeypatch.setenv("HOME", "/dev/null")
+    monkeypatch.delenv("XDG_CONFIG_HOME", False)
+    monkeypatch.chdir(tmp_path)
+    # For %packager
+    old = rpm.expandMacro("%packager")
+    with patch("specfile.changelog._getent_name", return_value=""):
+        try:
+            rpm.delMacro("packager")
+            yield
+        finally:
+            if old != "%packager":
+                rpm.addMacro("packager", old)
+
+
+@pytest.fixture
+def set_packager_env(monkeypatch: pytest.MonkeyPatch) -> str:
+    packager = "Patty Packager <patty@packager.me>"
+    monkeypatch.setenv("RPM_PACKAGER", packager)
+    return packager
+
+
+@pytest.fixture
+def set_packager_git(monkeypatch: pytest.MonkeyPatch, tmp_path: Path) -> Iterable[str]:
```

Comment:
The return type here doesn't seem right.

Reply:
You're right. It should be `str`.